### PR TITLE
added images and update manifest

### PR DESCRIPTION
**Pull request by xXSwainXx**
## Type of change

- [ ] Add new directory for a server
- [x] Update directory of a server
- [ ] Documentation Update

## Checklist

- [x] I have read the [README](https://github.com/LabyMod/server-media/blob/master/README.md) doc
- [x] I have compressed the images appropriately (e.g. on https://tinypng.com)
- [x] I have created the manifest.json with the required values

### CODE REVIEW

File: minecraft_servers/pluginstube/manifest.json [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  "server_name": "pluginstube",
+  "nice_name": "PluginStube",
+  "direct_ip": "pluginstube.net",
+  "supported_languages": [
+     "de"
+  ],
+  "social": {
+    "web": "https://pluginstube.net",
+    "web_shop": "https://shop.pluginstube.net",
+    "web_support": "https://pluginstube.net/support",
```
Comment (JakobBruening) [inline minecraft_servers/pluginstube/manifest.json:11]: Page not found.

**Review by JakobBruening — CHANGES_REQUESTED** [review]
Please compress your images.

File: minecraft_servers/pluginstube/manifest.json (hunk 1 above)
Comment (xXSwainXx) [reply minecraft_servers/pluginstube/manifest.json:11]: Hey i have fixed the https://pluginstube.net/support domain

and compressed the background image ;)

**Review by JakobBruening — APPROVED** [review]
Thank you :)

**Pull request merged**